data/table.txt

```diff
@@ -8123,13 +8174,14 @@
 勱 劢 1 1 0 0 1 0 0 0 0 tbks tbks,xxtbk NA 7371
 勲 勋 1 0 0 0 1 0 0 0 0 hsf hsf NA 0
 勳 勋 1 1 0 0 1 0 0 0 0 hfks hfks NA 16461
-勴 勴 1 1 0 0 1 0 0 0 0 ypks xypks,ypks NA 5376
+勴 勴 1 1 0 0 1 0 0 0 0 ypks ypks,xypks NA 5376
```

Comment:
@yookoala [said](https://github.com/Cangjians/libcangjie/commit/d9d2e686ce6776ec67e89b4256dce42359597d84#r29099657):

> "xypks" returns nothing on MacOS.

---

For `ypks`, we have the following characters:

```
虏 虏 1 0 0 0 1 0 0 0 0 ypks ypks NA 0
勴 勴 1 1 0 0 1 0 0 0 0 ypks ypks,xypks NA 5376
```

Isn't this a case of x-disambiguation?

Reply:
So Wan Leung just said that Mac OS X uses the short-lived Cangjie version 4, not version 5. :open_mouth:

Discussing this with @yookoala, we decided to ignore all those comments about Mac.